pkg/plugin/framework/server_mux.go

```diff
@@ -75,3 +80,18 @@ func (m *serverMux) getHandler(name string) (interface{}, error) {
 
 	return m.handlers[name], nil
 }
+
+func validPluginName(name string, existingNames []string) bool {
+	name = strings.ToLower(name)
+	tokens := strings.Split(name, "/")
+	if len(tokens) <= 1 || tokens[0] == "" || tokens[1] == "" {
```

Comment:
I think `velero.io/my-plugin/third-field` would pass here. How strict do we want to be? Do we want to have it be exactly 1 `/` and exactly 2 strings?

Reply:
I think the requirement should be `at least one`. @skriss wdyt?

Reply:
Yeah, I think we want to limit to exactly one slash. Ideally we validate that the first part is an FQDN as well (I *think* that spec is valid characters of a-z, 0-9, `.`, `-`, max total length 253, max length between `.`'s 63).